3-extra/1-factorial.js

```diff
@@ -10,6 +10,11 @@
 
 function factorial(input) {
     // TODO
+    let sum = 1
+    for (i = 1; i <= input; i++){
```

Comment:
Don't forget to initialise `i` with `let`

Reply:
Done